frontend/src/theme/components/Tabs.ts

```diff
@@ -0,0 +1,116 @@
+import {
+  ChakraTheme,
+  ComponentMultiStyleConfig,
```

Comment:
Focus styles seem to be missing, might want to ask @pearlyong to update them

Reply:
Okay, I've left a comment in figma!